packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -38,6 +38,29 @@ const Duration _kExpand = Duration(milliseconds: 200);
 /// ** See code in examples/api/lib/material/expansion_tile/expansion_tile.0.dart **
 /// {@end-tool}
 ///
+/// ## Troubleshooting
+///
+/// ### Why do I need to provide unique PageStorageKey(s) to scrollable(s) inside [ExpansionTile]
```

Comment:
This title basically describes a solution. When trouble-shooting you usually start with a problem, though. Maybe state what the problem is in the headline?